arch/x86/kernel/cpu/resctrl/rdtgroup.c

```diff
@@ -3495,6 +3510,8 @@ static int rdtgroup_mkdir_ctrl_mon(struct kernfs_node *parent_kn,
 
 out_del_list:
 	list_del(&rdtgrp->rdtgroup_list);
+out_rmid_free:
+	mkdir_rdt_prepare_rmid_free(rdtgrp);
 out_id_free:
```

Comment:
at upstream patch they changed the out_id_free to out_closid_free. Reference https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/arch/x86/kernel/cpu/resctrl/rdtgroup.c?id=311639e9512bb3af2abae32be9322b8a9b30eaa1

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev